hubspot3/tickets.py

```diff
@@ -54,10 +59,10 @@ def get(self, ticket_id: str, include_deleted: bool = False, **options) -> Dict:
         options.update({"params": params})
 
         return self._call(
-            "objects/tickets/{}".format(ticket_id), method="GET", **options
+            "objects/tickets/{}".format(ticket_id), method="GET", properties=properties, **options
         )
 
-    def get_all(self, limit: int = -1, **options) -> list:
+    def get_all(self, properties=["subject", "content", "hs_pipeline", "hs_pipeline_stage", "hs_pipeline"], limit: int = -1, **options) -> list:
```

Comment:
This line could be dangerous - using a mutable value as a default for a kwarg [can lead to unintentional behavior](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).

Normally, we'd do something more like this:
```python
def get_all(self, limit: int = -1, properties: List[str] = None, **options) -> list:
    # some code
    properties = properties or ["subject", "content", "hs_pipeline", "hs_pipeline_stage", "hs_pipeline"]
```